config/environments/development.rb

```diff
@@ -38,4 +38,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.delivery_method = :smtp
```

Comment:
Duplicate `delivery_method` config - this is already set in the mail_setup.rb initializer script.